services/osparc-gateway-server/tests/integration/conftest.py

```diff
@@ -107,7 +107,6 @@ async def local_dask_gateway_server(
     c.Proxy.address = f"{get_this_computer_ip()}:0"  # type: ignore
     c.DaskGateway.authenticator_class = "dask_gateway_server.auth.SimpleAuthenticator"  # type: ignore
```

Comment:
@mrnicegyu11 If time permits, could you maybe start having a look at how Kerberos works? All we need is one user with a password.